services/webhook/elder_eval/harvest_bot_findings.py

```
@@ -0,0 +1,389 @@
+# Network main (no tests import below this line; mirrors sast_benchmark)
+# --------------------------------------------------------------------------- #
+
+_DEFAULT_REPOS = (
+    "grug", "digital-ledger", "claude-stuff", "infra", "infra-public",
+    "macchina", "grugthink", "brother-claudius", "conducted", "holdfast",
+    "meow-now", "vroom-vroom", "gemini-plugin-cc", "aws-solutions-architect-study",
+)
+_PAGE_CAP = 30  # 3000 comments/repo/endpoint - far past the trial volume
+
+
+def _paged(client: Any, url: str, *, params: dict[str, Any]) -> Iterable[dict[str, Any]]:
```

Ruff (0.15.20)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `client`

(ANN401)